package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dependency-check",
-  "version": "2.5.1",
+  "version": "2.5.2",
```

Comment:
usually the maintainer does this, as they have to decide how to interpret semver for the changes. in this case you incremented the patch version, but what I will end up doing is most likely incrementing the minor version as you've added new APIs. I would just change this back to 2.5.1 and let me increment the version when I publish the new stuff to npm